model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -83,18 +83,117 @@ elif [[ "$OSTYPE" == "darwin"* ]]; then
+                        sys.path.append(os.path.join(\"$SCRIPTDIR\", os.pardir)); \
+                        from mo.utils.version import extract_release_version; \
+                        print(\"{}.{}\".format(*extract_release_version()))")"
+}
+
+find_ie_bindings() {
+    python_executable="$1"
+    requires_sudo="$2"
+
+    if ! check_ie "$python_executable"; then
+        # check pip installed openvino package
+        if check_ov_package "$python_executable" "$requires_sudo"; then
+            echo "[ WARNING ] Existing OpenVINO version doesn't work as expected."
+            return 1
+        fi
+
+        # try to find OpenVINO version that matches MO version
+        if extract_release_from_version "$python_executable"; then
+            if [[ $version == "None.None" ]]; then
+                echo "[ WARNING ] Can not extract release version from ModelOptimizer version. The latest OpenVINO version will be installed (may be incompatible with current ModelOptimizer version)"
+            elif install_ov "$python_executable" "$requires_sudo" "$version"; then
+                if check_ie "$python_executable"; then
+                    return 0
+                fi
+                echo "[ WARNING ] Installed OpenVINO version doesn't work as expected...Uninstalling"
+                uninstall_ov "$python_executable" "$requires_sudo"
+                return 1
+            else
+                echo "[ WARNING ] Can not find OpenVINO version that matches ModelOptimizer version. The latest OpenVINO version will be installed (may be incompatible with current ModelOptimizer version)"
+            fi
+        else
+            echo "[ WARNING ] Can not detect ModelOptimizer release version. The latest OpenVINO version will be installed (may be incompatible with current ModelOptimizer version)"
+        fi
+
+        #install the latest OpenVINO pip version
+        if install_latest_ov "$python_executable" "$requires_sudo"; then
+            if check_ie "$python_executable"; then
+                return 0
+            else
+                echo "[ WARNING ] Installed OpenVINO version doesn't work as expected...Uninstalling"
+                uninstall_ov "$python_executable" "$requires_sudo"
+                return 1
+            fi
+        else
+            echo "[ WARNING ] No OpenVINO version is available for installation"
+            return 1
+        fi
+    fi
+
+    return 0
+}
 
 if [[ $V_ENV -eq 1 ]]; then
     "$python_binary" -m venv "$SCRIPTDIR/../venv${postfix}"
     source "$SCRIPTDIR/../venv${postfix}/bin/activate"
-    "$SCRIPTDIR/../venv${postfix}/bin/$python_binary" -m pip install -r "$SCRIPTDIR/../requirements${postfix}.txt"
+    venv_python_binary="$SCRIPTDIR/../venv${postfix}/bin/$python_binary"
+    $venv_python_binary -m pip install -r "$SCRIPTDIR/../requirements${postfix}.txt"
+    if ! find_ie_bindings "$venv_python_binary" false; then
+        echo "[ WARNING ] Please build IneferenceEngine Python bindings from source"
```

Comment:
This is already a solution, but we haven't described the issue itself. It may happen that something is wrong with installation and no need to build python bindings.

Reply:
During IE search we show impor errors and in the end we suggest some additional steps what customer can do if for example current OV doesn't work or even doesn't exists.